irr/include/SColor.h

```diff
@@ -104,11 +107,12 @@ const c8 *const ColorFormatNames[ECF_UNKNOWN + 2] = {
 		"R16",
 		"R16G16",
 		"D16",
+		"D24",
 		"D32",
 		"D24S8",
 		"UNKNOWN",
 		0,
-	};
+};
```

Comment:
This what Irrlicht code usually does:

```cpp
//! Names for ECOLOR_FORMAT types
const c8 *const ColorFormatNames[ECF_UNKNOWN + 2] = {
		"A1R5G5B5",
		...
		"D32",
		"D24S8",
		"UNKNOWN",
		0,
	};
```

This would be okay too (but inconsistent with the rest of the Irrlicht code):
```cpp
const c8 *const ColorFormatNames[ECF_UNKNOWN + 2] = {
	"A1R5G5B5",
	...
	"D32",
	"D24S8",
	"UNKNOWN",
	0,
};
```

But this version doesn't make sense to me:

```cpp
//! Names for ECOLOR_FORMAT types
const c8 *const ColorFormatNames[ECF_UNKNOWN + 2] = {
		"A1R5G5B5",
		...
		"D32",
		"D24S8",
		"UNKNOWN",
		0,
};
```

Reply:
I'll re-add it